src/main/java/ca/softwarespace/qiyanna/dataaggregator/repositories/sql/SQLAggregatorInfoRepositoryImpl.java

```diff
@@ -0,0 +1,51 @@
+package ca.softwarespace.qiyanna.dataaggregator.repositories.sql;
+
+import static ca.softwarespace.qiyanna.dataaggregator.models.generated.tables.AggregatorInfo.AGGREGATOR_INFO;
+
+import ca.softwarespace.qiyanna.dataaggregator.models.dto.AggregatorInfoDto;
+import ca.softwarespace.qiyanna.dataaggregator.models.exceptions.RecordNotFoundException;
+import ca.softwarespace.qiyanna.dataaggregator.models.generated.tables.records.AggregatorInfoRecord;
+import ca.softwarespace.qiyanna.dataaggregator.repositories.sql.interfaces.SQLAggregatorInfoRepository;
+import org.jooq.DSLContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+@Repository
+public class SQLAggregatorInfoRepositoryImpl implements SQLAggregatorInfoRepository {
+
+  private final DSLContext dsl;
+
+  @Autowired
+  public SQLAggregatorInfoRepositoryImpl(DSLContext dsl) {
+    this.dsl = dsl;
+  }
+
+  @Transactional
+  @Override
+  public AggregatorInfoDto update(AggregatorInfoDto dto) throws RecordNotFoundException {
```

Comment:
Not sure if the methods need to be transactional, working with alot of data, will cause it to slow down